Britannia/events/BTWK_storyline_events.txt

```diff
@@ -4,96 +4,676 @@
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					has_primary_title = title:k_cynwidion
+				}
+			}
+			title:k_gwent.holder = {
+				marry = character:19
+			}
+		}
+		else_if = {
+			limit = {
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					is_primary_heir_of = title:k_cynwidion.holder
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+	################### Demetia ###################
+	option = { 
+		name = BTWK_story_event.0104.h
+		trigger = {
+			any_in_global_list = {
+				variable = char_is_norwenna_wedding_prospect
+				OR = {
+					has_primary_title = title:k_demetia
+					AND = {
+						is_primary_heir_of = title:k_demetia.holder
+						holds_any_storyline_britannia_kingdoms = no
+						}
+					}
+				}
+			}
+		if = {
+			limit = { 
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					has_primary_title = title:k_demetia
+				}
+			}
+			title:k_gwent.holder = {
+				marry = character:19
+			}
+		}
+		else_if = {
+			limit = {
+				any_in_global_list = {
+					variable = char_is_norwenna_wedding_prospect
+					is_primary_heir_of = title:k_demetia.holder
+				}
+			}
+			title:k_gwent.holder.primary_heir = {
+				marry = character:19
+			}
+		}
+		character:1	= {
+			trigger_event = BTWK_story_event.0105
+		}
+	}	
+}
+
+BTWK_story_event.0105	= { # Guardian Selection Event
+
+# SELECTION OF GUARDIANS QUITE IRRELEVANT IN THE STORY. COULD GIVE PLAYER THE OPTION TO CHOOSE FROM REMAINING SUITORS, OR LEAVE IT AS CANON.
+# IF HAVE LIST OF ALL PRESENT, ADD TRUCES BETWEEN EACH KING TITLE HOLDER. 
+# ADD ALLIANCE TO NORWENNA SPOUSE, OR TO CHARACTER THE SPOUSE IS PRIM HEIR TO
+
+	type = character_event
+	title = BTWK_story_event.0105.t
+	desc = BTWK_story_event.0105.desc
+	theme = realm 
+
+	right_portrait = {
+		character = character:1  # Uther
+		animation = personality_honorable
 	}	
 	lower_left_portrait = {
-		character = character:5  # Mordred ap Mordred
+		character = character:5  # Mordred
 		animation = no
 	}
 	lower_center_portrait = {
-		character = character:2  # Merlin
+		character = character:14  # Tewdric
 		animation = no
 	}
 	lower_right_portrait = {
 		character = character:3  # Arthur
 		animation = no
 	}
-	
+
 	option = {
-		name = BTWK_story_event.0001.a
-		set_global_variable = utherillness
+		name = BTWK_story_event.0105.a
+		character:1 = {
+			trigger_event = { 
+				id = BTWK_story_event.0106
+				days = {25 50}
+			}
+		}
+	}
+}
+
+BTWK_story_event.0106 = {  
+	type = character_event
+	title = BTWK_story_event.0106.t
+	desc = BTWK_story_event.0106.desc
+	theme = skull 
+
+	# More fitting music cue/sounds needed
+	right_portrait = {
+		character = character:1  # Uther
+		animation = personality_honorable
+	}	
+	option = {
+		name = BTWK_story_event.0106.a
 		character:1 = {
 			add_trait = pneumonic
```

Comment:
Can you check back in the books how Uther actually dies? From my memory he actually has pneumonia at start, gets a little better, and then dies the next fall anyways, but I don't remember how that's described.

Reply:
I don't recall it being described to much detail at all when I read through the first few chapters the other week.  I also believe his death reason wasn't described, and it just said he died. I'll read through and check when I have more time. 

I had removed the pnemonia from the history files to test the events. I'll re-add it, but I could add some checks to circumvent Glevum if Uther dies before the council, and still have Arthur inherit his titles. Alternatively, give him a temporary health bonus until Glevum is over so he doesn't die before it. I know at the minute he gets the near death popup on game start.

Reply:
Probably best to do both - even if we give him the health boost, he might die randomly from somethig else.